WordFrequency/WordCountPair.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace WordFrequency
+{
+    public class WordCountPair
```

Comment:
if Word may be better?

Reply:
I vote for WordCountPair, it will reveal the count function in this class